includes/functions/functions_lookups.php

```diff
@@ -311,20 +311,20 @@ function zen_lookup_admin_menu_language_override(string $lookup_type, ?string $l
             break;
         case 'configuration_group_title':
             $str = $lookup_key;
-            $str = preg_replace('/[\s ]+/', '_', $str);
```

Comment:
Note to self: The `[\s ]` pattern was introduced in #6584 with v2.1.0
So backward-compatibility of including the `-` is only impacted between v2.1.0 and v2.2.0